index.html

```diff
@@ -34,5 +34,154 @@ <h2 class="welcoming">Hey there. I’m Anas
          </ul>
        <div class="angle"> <img src="images/angle.png" alt=""></div> 
    </section>
+
+   <section class="works">
+   
+   <h3 class="Section-title">My Recent Works</h3>
+      <div class="card card1">
+        <img src="images/ImgPlaholder.png" alt="">
+         <div>
+            <p>Multi-Post Stories Gain+Glory</p>
+            <ul>
+               <li>Ruby on rails</li>
+               <li>css</li>
+               <li>JavScript</li>
+               <li>html</li>
+            </ul>
+            <button type="button">See Project</button>
+         </div>
+
```

Comment:
- Please for each button please remove the dark borderline, so that it looks exactly like the Figma design. 
Hint: Add  `border:none` to the button class

##### Figma Design 
![Screenshot from 2022-06-01 17-48-09](https://user-images.githubusercontent.com/58771507/171457923-3f7cf253-db67-4d72-bea6-75240d5279ac.png)

##### Your Design
![Screenshot from 2022-06-01 17-47-37](https://user-images.githubusercontent.com/58771507/171457936-fa9e7a61-5b50-4a4b-b48c-bde7f386bf11.png)

Reply:
one moment